src/middleware/authentication.ts

```diff
@@ -16,12 +16,10 @@ export async function authenticationMiddleware(
     const formId = request.params.formId;
 
     if (!accessToken) {
-      // Authorization header is missing
```

Comment:
The recently added JSON message does pretty much explains the check we are doing here.